lib/licensee/matchers/dist_zilla_matcher.rb

```diff
@@ -0,0 +1,18 @@
+module Licensee
+  module Matchers
+    class DistZilla < Package
+      attr_reader :file
+
+      LICENSE_REGEX = /
+        ^license\s*=\s*([a-z\-0-9\.]+)
```

Comment:
I believe this can be simplified to a single line. `/^license\s*=\s*([a-z\-0-9\.]+)/i`.